ImageGalleryClient/ImageGalleries.csproj.user

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You don't want to commit .user files to repos.  Should be removed using an entry in `.gitignore`